rocky/account/views/account.py

```diff
@@ -13,31 +15,36 @@ class PageActions(Enum):
     WITHDRAW_ACCEPTANCE = "withdraw_acceptance"
 
 
-class AccountView(OrganizationView, DetailView):
-    template_name = "account_detail.html"
-    context_object_name = "member"
-
-    def get_object(self):
-        return OrganizationMember.objects.get(user=self.request.user, organization=self.organization)
-
+class OOIClearanceManager(OrganizationView, TemplateView):
```

Comment:
This is a Mixin, not a Manager. That also means it shouldn't inherit from the other views.